### DISCUSSION TITLE
A mark or color for the frames that are clip

### DISCUSSION THREAD

**Discussion by amoraleite** [keiwando/toonsquid#520] [Ideas]
My workflow is a mix of frame by frames, clips and key frames... but sometimes I would like to do a frame and after a clip. So I would like to visually identify the frames that are clips... is a small idea.

**Comment by andydjames** [comment]
It would be good to be able to mark individual frames as Keyframes, Extremes, Inbetweens so that marks move as the frames move in the timeline. If you use markers, they don't move as you insert new inbetweens, for example.
